config/initializers/inflections.rb

```diff
@@ -18,15 +18,18 @@
   inflect.acronym 'GIDS'
   inflect.acronym 'GI'
   inflect.acronym 'HCA'
+  inflect.acronym 'HTTP'
```

Comment:
Zeitwerk::NameError: expected file ~/vets-api/lib/vets/type/http_date.rb to define constant Vets::Type::HttpDate, but didn't (Zeitwerk::NameError)